src/components/AddPlaidBankAccount.js

```diff
@@ -122,10 +126,15 @@ class AddPlaidBankAccount extends React.Component {
      * @param {String} index
      */
     selectAccount(index) {
+        if (index === undefined) {
```

Comment:
```suggestion
        if (_.isNull(index)) {
```
This will no longer ever be undefined, correct?